galite-core/src/main/kotlin/org/kopi/galite/util/Printer.kt

```diff
@@ -17,6 +17,31 @@
  */
 package org.kopi.galite.util
 
+import java.awt.PrintJob
+
 interface Printer {
```

Comment:
add docs
/**
 * An interface that defines printer devices
 *
 * Such printers may be remote printers, local printers, print-to-file, fax
 * mailer or whatever
 */